packrd/packed-packr.go

```diff
@@ -0,0 +1,34 @@
+	})
+	if err != nil {
+		panic(err)
+	}
+	g.DefaultResolver = hgr
+
+	func() {
+		b := packr.New("configBox", "./templates")
+		b.SetResolver("integration_endpoints_user_config_schema.json", packr.Pointer{ForwardBox: gk, ForwardPath: "0260f8a42944adbc9bffd306f2c2c467"})
+		b.SetResolver("integrations_user_config_schema.json", packr.Pointer{ForwardBox: gk, ForwardPath: "66afd4a4db0cb1c31e4889c3d0608a0a"})
+		b.SetResolver("service_user_config_schema.json", packr.Pointer{ForwardBox: gk, ForwardPath: "e289475498b6f5ecacf83f51cf68d371"})
+	}()
+
+	return nil
+}()
```

Comment:
At least in upstream - this folder is gitignored. 

I don’t know best practices here - but I assume this file will churn rather randomly. 

https://github.com/aiven/terraform-provider-aiven/blob/master/.gitignore#L33

Reply:
This will only change when the json files change - it's not checked into github on that repo as they build it into the binary - we need it checked into ours as we check that everything is without change before we build the binaries :D